package.json

```diff
@@ -16,11 +16,11 @@
     "create_db": "cross-env NODE_ENV=test node dist/create_db.js",
     "delete_db": "cross-env NODE_ENV=test node dist/delete_db.js",
     "build": "gulp",
-    "lint": "esw -c .eslintrc *.js server config . --color --fix",
+    "lint": "esw -c .eslintrc --color --fix .",
     "lint:watch": "yarn lint -- --watch",
     "beautify": "esw -c .eslintrc *.js server config --color --fix",
     "precommit": "yarn lint && yarn test",
-    "test": "gulp && yarn create_db && cross-env NODE_ENV=test ./node_modules/.bin/mocha --ui bdd --reporter spec --colors --compilers js:babel-core/register server/tests --recursive || yarn delete_db",
+    "test": "gulp && yarn create_db && cross-env NODE_ENV=test ./node_modules/.bin/mocha --ui bdd --reporter spec --colors --compilers js:babel-core/register server/tests --recursive && yarn delete_db || yarn delete_db",
```

Comment:
Why is there multiple calls to delete_db here ?

Reply:
See https://github.com/amida-tech/amida-messaging-microservice/pull/67#discussion_r220583818
That is because of if its just an `||` it doesnt run when all the tests pass. When it is `&&` it doesnt run when all the tests dont pass. With both, it will run in both situations - open to suggestions on better ways.

Reply:
this is actually the best cross platform solution there is for this since `;` does not work properly on windows.